src/scss/components/optional/_modal.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Components > Item Selector
+ * =================================
+ * Used on menu pages to display a modal to the user. Styled as a normal modal on
```

Comment:
"Styled as a normal modal on " should there be something else to finish the sentence? :)

Reply:
Was a mistake, have removed now.